styles/components/_week-select.scss

```diff
@@ -0,0 +1,61 @@
+.cmp-week-select {
+  &__label {
+    font-size: font-size("lg");
+    letter-spacing: -0.02em;
+
+    @media (min-width: $sm-screen-width) {
+      font-size: font-size("xl");
+    }
+
+    @media (min-width: $md-screen-width) {
+      font-size: font-size("3xl");
+    }
+  }
+
+  &__menu {
+    appearance: none;
+    background:
+      transparent
+      url("data:image/svg+xml;utf8,<svg viewBox='0 0 33 19' width='14.67' height='8.44' xmlns='http://www.w3.org/2000/svg' fill='none'><path stroke='%23fff' stroke-width='2.9' d='M1.1 2 17.1 17.1M31.1 2 15 17.1'/></svg>")
+      no-repeat;
+    background-position: right 0.5rem top 60%;
+    border: none;
+    color: color("white");
+    display: inline-block;
+    font-family: font-family("sans-extended");
+    font-size: font-size("lg");
+    font-weight: 400;
+    letter-spacing: -0.02em;
+    margin-top: space("2xs");
+    padding-right: space("sm");
```

Comment:
I'd suggest making the `padding-right` value at least the same as the width of the drop-drown indicator svg (2.1rem), just to avoid the overlap with the text that occurs in some weeks.

<img width="774" alt="Screen Shot 2022-06-09 at 9 58 24 AM" src="https://user-images.githubusercontent.com/2685062/172866007-8c24b5db-f598-4944-9578-0f23f9750d43.png">